### PR TITLE
Simplify fieldservice_stock

### PR DESCRIPTION
**Pull request by brian10048**
#674 

The module fieldservice_stock has grown to include extra dependencies and features that may not be necessary for all use cases.  This PR will help to:

- [x] remove `stock_request_*` dependencies and simplify to only depend on Odoo `stock`
- [x] remove FSM Equipment related functions and move to own module
- [x] add tests for all affected modules
- [x] update readmes accordingly

### CODE REVIEW

File: fieldservice_delivery/__manifest__.py [hunk 1]
```diff
@@ -1,16 +1,16 @@
-# Copyright (C) 2018 - TODAY, Brian McMaster
+# Copyright (C) 2018, Open Source Integrators
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 {
     'name': 'Field Service - Delivery',
     'summary': 'Select delivery methods and carriers on Field Service orders',
-    'version': '12.0.1.0.0',
+    'version': '12.0.1.1.0',
```
Comment (dreispt) [inline fieldservice_delivery/__manifest__.py]: Version numbers will be bumped by bot upon merge, it is best to not change them in the PRs

Comment (brian10048) [reply fieldservice_delivery/__manifest__.py]: 👍 Updated

**Comment by brian10048** [comment]
@hparfr are you able to review this PR?

I know it is somewhat large, but I also tried to break up the commits for easy review.  There is no new functionality, just moving into individual modules and then adding tests

**Review by hparfr — APPROVED** [review]
Quick code review.

**Review by smangukiya — APPROVED** [review]
LGTM.

**Review by marcelsavegnago — APPROVED** [review]
LGTM

**Comment by max3903** [comment]
@brian10048 Next time you refactor like this, please provide the migration script to cleanup views. Thank you.

**Comment by brian10048** [comment]
My apologies @max3903. I hope I didn't cause your team too much trouble

Would using rename_xmlids from openupgradelib have been correct?

**Pull request merged**